<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWindow</class>
 <widget class="QMainWindow" name="RoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
border-image: url(:/image/2/aaaa.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="ledWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>19</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2/bulb_close_1.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ledPushButton1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ledPushButton2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="ledWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>19</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2/bulb_close_1.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="fanWidget" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2/fan_close.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fanPushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="doorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2/door_close2.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
